packages/iota/src/models/payloads/IMilestonePayload.ts

```diff
@@ -43,14 +44,14 @@ export interface IMilestonePayload extends ITypeBase<1> {
     nextPoWScoreMilestoneIndex: number;
 
     /**
-     * The public keys.
+     * Hex-encoded binary data with 0x prefix.
```

Comment:
Maybe this should read something like "The metadata." or "The milestone metadata.", to stay consistent with other fields.